src/core/js/adapt.js

```diff
@@ -106,9 +106,11 @@ define([
         }
 
         var navigationHeight = $(".navigation").outerHeight();
+        // prevent scroll issue when component description aria-label coincident with top of component
+        var ariaLabelBuffer = $(selector.is('.component')) ? 5 : 0;
```

Comment:
I can't actually replicate the original issue in IE8, while the fix works for me in Firefox & Edge.